cabal.project

```diff
@@ -94,9 +94,9 @@ source-repository-package
 
 source-repository-package
   type: git
-  location: https://github.com/maestro-org/haskell-sdk
-  tag: deb1e133cd7de989e9e85c2b44e9361f52e93776
-  --sha256: sha256-k4OoAAUFdYkKDSTDeM2dwC/DrJrPotiz0EIeU0xccXY=
+  location: https://github.com/alegadea/haskell-sdk
+  tag: d916e60c0f4eef5d9d51385ada99b014b86fae7e
+  --sha256: sha256-nZHSNhulnbaXhRpwrQbKzVLPPrJpFiTmz18PArLhgNs=
```

Comment:
Once we merge this PR (https://github.com/maestro-org/haskell-sdk/pull/36), we should update here.

Reply:
Done @sourabhxyz